source/clear-linux/concepts/security.rst

```diff
@@ -1,7 +1,7 @@
 .. _security:
 
 Clear Linux\* OS Security 
```

Comment:
Please replace "Clear Linux\* OS" with CL-ATTR (title and first paragraph) :)

Reply:
Also - under "Compiled with secure options" section, replace "Clear Linux" in text with CL substitution.